corazon_runner/utils.py

```diff
@@ -0,0 +1,124 @@
+import glob
+import os
+import subprocess
+import time
+import types
+
+from corazon_runner.constants import LOCAL_DATA_PATH
+from corazon_runner.datatypes import TESSLightCurveFile
+
+
+def locate_and_resolve_tess_datums(input_dir: str) -> types.GeneratorType:
```

Comment:
This function searches the local file-system for filenames that match expected input. ( spoc & qlp files ? )

After locating files that exist, the files are then returned out of the function as a TESSLightCurveFile datatype.